engine/gamesys/src/gamesys/scripts/script_resource.cpp

```diff
@@ -2573,6 +2718,7 @@ static void LuaInit(lua_State* L)
 
     SETGRAPHICSCONSTANT(TEXTURE_TYPE_2D);
     SETGRAPHICSCONSTANT(TEXTURE_TYPE_CUBE_MAP);
+    SETGRAPHICSCONSTANT(TEXTURE_TYPE_2D_ARRAY);
```

Comment:
Added because even if we don't support creating dynamic array textures (yet), we can still get the info from a texture resource that has been created as an array texture!